Gems/ROS2/Code/CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ ly_add_target(
             Gem::LmbrCentral.API
 )
 
-target_depends_on_ros2_packages(${gem_name}.Static rclcpp builtin_interfaces std_msgs sensor_msgs nav_msgs tf2_ros ackermann_msgs gazebo_msgs)
+target_depends_on_ros2_packages(${gem_name}.Static rclcpp builtin_interfaces std_msgs sensor_msgs nav_msgs tf2_ros ackermann_msgs gazebo_msgs vision_msgs)
```

Comment:
Is vision messages needed?

Reply:
Yes, It's used to publish  vision_msgs::msg::LabelInfo segmentationClasses